lib/modulesync.rb

```diff
@@ -78,6 +80,7 @@ def self.run(args)
           end
         end
         files_to_manage -= files_to_delete
+        next if options[:offline]
```

Comment:
I think we do still want the update_noop to happen in offline mode (it just prints the diffs to stdout)